manifests/profiles/PROFILES.md

```diff
@@ -0,0 +1,18 @@
+# Kubernetes Vendor Optimized Profiles
+
+Vendor optimized profiles are approved in [Istio RFC: Kubernetes Vendor Optimized Profiles](https://docs.google.com/document/d/1N_yC9vi4Jt3auMLLSOkavQb6qgbJKr-Mp4kUL3GOmEc/edit#heading=h.xw1gqgyqs5b)
+
+## Add your vendor optimized profile
+
+If you wish to add a Kubernetes optimized profile, please join the Environments
+WG meeting with your two lead maintainers for the profile. The working group
+will provide further instructions. Additionally, please read the RFC requirements.
+
+## Vendor optimized profiles
+
++----------------------+---------------+-----------------+------------------+
```

Comment:
This table does not render correctly

Reply:
this is now resolved. Thanks @jwendell !